app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Show User Info for a given user</h1>
\ No newline at end of file
+<div id="userTemplate">
+   <%= render partial: "shared/single_user", locals: { user: @user } %>
+   <div id="bio">
+   <h3>Bio</h3>
+   <p><%= @user.bio %></p>
+   </div>
+<% @user.posts.each do |post| %>
+     <%= render partial: "shared/single_post", locals: { post: post } %>
```

Comment:
- [ ] It should use `#recent_post` method to render the user's 3 most recent post. At the moment you are rendering all user's posts. Please kindly make sure the user can only see the 3 most recent posts.